cmake/sdp_asm_install.cmake

```diff
@@ -10,12 +10,12 @@ function(asm_install)
     get_filename_component(asm_filename ${hrt_src} NAME_WE)  # filename without extension
     get_filename_component(src_dir ${hrt_src} DIRECTORY)
 
-    file(RENAME ${asm_filename}-temp.s ${src_dir}/${asm_filename}.s RESULT rename_result)
+    file(RENAME ${asm_filename}-${target_soc}-temp.s ${src_dir}/${asm_filename}-${target_soc}.s RESULT rename_result)
```

Comment:
`target_soc` is a required argument to the CMake script, please ensure proper validation.